src/genie/libs/parser/iosxe/show_bootflash.py

```diff
@@ -0,0 +1,61 @@
+''' show_bootflash.py
+
+IOSXE parsers for the following show commands:
+    * show bootflash:
+'''
+
+# Python
+
+import re
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+# parser utils
+from genie.libs.parser.utils.common import Common
+
+# =================
+# Schema for:
+#  * 'show bootflash:'
+# =================
+
+class ShowBootflashSchema(MetaParser):
+    """Schema for show bootflash:."""
+    schema = {
+        'available': str,
+        'used': str
+    }
+
+# =================
+# Parser for:
+#  * 'show bootflash:'
+# =================
+class ShowBootflash(ShowBootflashSchema):
+    """Parser for show bootflash:"""
+
+    cli_command = 'show bootflash:'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        #         available used
+        p1 = re.compile(r"(?P<available>\d+)\s+bytes available\s+\((?P<used>\d+)\s+bytes used\)")
+
+
+        # initial variables
+        ret_dict = {}
+
+        for line in out.splitlines():
+            line_strip = line.strip()
+            m = p1.match(line_strip)
```

Comment:
could you put the matched line above here as well, same as above comment, thanks.